calculation/sha_calc/sha_calc/directivity/bea20/directivity.py

```diff
@@ -117,45 +127,53 @@ def compute_fault_directivity(
         The magnitude of the fault
     rake: float
         The rake of the fault
-    period: float
-        The period for fdi to extract from the bea20 models fd
+    periods: List[float], optional
+        The periods to calculate for the bea20 model's fD
+        If not set then will be all the default pSA periods for GMHazard
+    method: HypoMethod, optional
+        Method to place hypocentres across strike and dip
     """
     nominal_strike, nominal_strike2 = utils.calc_nominal_strike(lon_lat_depth)
 
     if hyp_down_dip == 1 and hyp_along_strike == 1:
-
-        # Gets the plane index of the hypocentre
-        plane_index = 0
-        for index, plane in enumerate(planes):
-            if plane["dhyp"] != -999.99:
-                plane_index = index
-                break
+        planes_list, plane_index = utils.set_hypocentres(
```

Comment:
I think this might get an error?
Checked that `utils.set_hypocentres()` function and with good story, return one of the distributions functions, it returns three different values. E.g, return planes_list, planes_index, weights for `even_grid()`, return planes_list, planes_index, [] for the rest? so we might need the third variable to set? as we did, _ if not needed.

Reply:
I believe this was solved with removing the if statement in this function (not doing different method of computing for 1 hypocentre vs many)

Reply:
My bad, even I got confused by reading my own comments :thinking: 

I could've made this simple and straightforward, I have a feeling that we would face a similar error to my other comments, something to do with `not enough values to unpack (expected 3, got 2)?`
(At least I left something useful within the comment above)
The function `utils.set_hypocentres` will always return three different values.
1. planes_list
2. planes_index
3. weights or []

So my question was whether we have to do something like
```python
# Also, is `planes_list` meant to be plural, planes and `plane_index` meant to be singular, plane?
planes_list, plane_index, weights = utils.set_hypocentres()....
```